operator/roles/csiscaleoperator/defaults/main.yml

```diff
@@ -62,6 +62,8 @@ state: present
 # Kubelet root directory path.
 kubeletRootDirPath: "{{ _csi_ibm_com_csiscaleoperator.spec.kubeletRootDirPath | default('/var/lib/kubelet') }}"
 
+csipspname: "{{ _csi_ibm_com_csiscaleoperator.spec.csipspname | default('ibm-spectrum-scale-csi-psp') }}"
```

Comment:
we can skip setting default if we want use is defined in template

Reply:
resolved in new change